### PR TITLE
Update @ew-did-registry@0.8.1-alpha.1122

### PR DESCRIPTION
**Pull request by JGiter**
### Contributor checklist

- [x] Breaking changes - check for any existing interfaces changes that are not backward compatible, removed method etc.
- [x] Documentation - document your code, add comments for method, remember to check if auto generated docs were updated.
- [x] Tests - add new or updated existed test for changes you made.
- [x] Migration guide - add migration guide for every breaking change.
- [x] Configuration correctness - check that any configuration changes are correct ex. default URLs, chain ids, smart contract verification on [Volta explorer](https://volta-explorer.energyweb.org/) or [EWC explorer](https://explorer.energyweb.org/).

### CODE REVIEW

**Review by nichonien — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -63,26 +63,27 @@
...
-    "@ew-did-registry/jwt": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/keys": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/proxyidentity": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/revocation": "0.8.1-alpha.1037.0",
+    "@ew-did-registry/claims": "^0.8.1-alpha.1122.0",
+    "@ew-did-registry/credentials-interface": "^0.8.1-alpha.1122.0",
+    "@ew-did-registry/did": "^0.8.1-alpha.1122.0",
+    "@ew-did-registry/did-document": "^0.8.1-alpha.1122.0",
+    "@ew-did-registry/did-ethr-resolver": "^0.8.1-alpha.1122.0",
+    "@ew-did-registry/did-ipfs-store": "^0.8.1-alpha.1122.0",
```
Comment (jrhender) [inline package.json:77]: I'm not sure we want to update `did-ipfs-store` yet, until we are confident that EW's IPFS cluster infrastructure is suitable. Let's discuss this further

**Review by jrhender — CHANGES_REQUESTED** [review]

File: e2e/utils/setUpIpfsCluster/docker-compose.yml
Comment (jrhender) [inline e2e/utils/setUpIpfsCluster/docker-compose.yml:1]: Is `setUpIpfsCluster` needed even if we've kept `"@ew-did-registry/did-ipfs-store"` as `"^0.7.1-alpha.816.0"`?

**Review by jrhender — APPROVED** [review]
A small question but it looks good to me 👍

File: e2e/utils/setUpIpfsCluster/docker-compose.yml
Comment (JGiter) [reply e2e/utils/setUpIpfsCluster/docker-compose.yml:1]: No, it is not used now. I kept it since we are moving to cluster anyway

**Pull request merged**